docs/DeveloperGuide.md

```diff
@@ -121,85 +121,85 @@ testers are expected to do more *exploratory* testing.</div>
 
-1. **Test case** `add pizza/34/340/eggs/dinner/www.food.com`<br>
-   * **Expected**: Mentioned recipe is added to the list of recipes. <br>
-   Console output providing a brief description of the added recipe <br>
+1. **Test case** `add pizza/34/340/eggs/dinner/www.food.com
+   * **Expected**: Mentioned recipe is added to the list of recipes.
+   Console output providing a brief description of the added recipe 
    and total number recipes present in list 
 
-2. **Test case** `add pizza/34/340/www.food.com`<br>
-   * **Expected**: Recipe is not added to the list due to format error.<br>
+2. **Test case** `add pizza/34/340/www.food.com`
+   * **Expected**: Recipe is not added to the list due to format error.
    Console output stating for the user to follow the correct format.
 
-3. **Test case** `add pizza/aa/aa/eggs/dinner/www.food.com`<br>
-   * **Expected**: Recipe is not added to list due to incorrect input types.<br>
-   Console output asking for user to use integer for fields of cook time <br>
+3. **Test case** `add pizza/aa/aa/eggs/dinner/www.food.com`
+   * **Expected**: Recipe is not added to list due to incorrect input types.
+   Console output asking for user to use integer for fields of cook time
    and calories
 
-4. **Test case** `add pizza/30/30/eggs/dinnerTest/www.food.com`<br>
-   * **Expected**: Recipe is not added to the list due to invalid meal categories.<br>
-   Console output asking for user select correct meal category from options of <br>
+4. **Test case** `add pizza/30/30/eggs/dinnerTest/www.food.com`
+   * **Expected**: Recipe is not added to the list due to invalid meal categories.
+   Console output asking for user select correct meal category from options of
    breakfast, lunch, dinner, appetizer and dessert
 
 ### Deleting a recipe
 
 1. **Prerequisites**: List all recipes using the `list` command. Multiple recipes in the list.
 
-2. **Test case**: `delete 1`<br>
-   * **Expected**: First recipe is deleted from the list. Console output <br>
+2. **Test case**: `delete 1`
+   * **Expected**: First recipe is deleted from the list. Console output 
    saying which recipe was deleted with a brief description.
 
-3. **Test case**: `delete 0`<br>
-   * **Expected**: No recipe is deleted due to index given being out of bounds. <br>
+3. **Test case**: `delete 0`
+   * **Expected**: No recipe is deleted due to index given being out of bounds.
    Console output tells you to input a number within the range of the recipe list numbers.
 
-4. **Test case**: `delete xyz`<br>
-   * **Expected**: No recipe is deleted. Console output tells you to input <br>
+4. **Test case**: `delete xyz`
+   * **Expected**: No recipe is deleted. Console output tells you to input
    an integer instead of other data types.
 
-5. **Test case**: `delete`<br>
-   * **Expected**: No recipe is deleted due to given index. Console output <br>
+5. **Test case**: `delete`
+   * **Expected**: No recipe is deleted due to given index. Console output
```

Comment:
If you breakline for Console output previously, please do so for the rest the section you wrote as well.